test/v8-to-istanbul.js

```diff
@@ -177,6 +191,8 @@ ${'//'}${'#'} sourceMappingURL=data:application/json;base64,${base64Sourcemap}
 
       v8ToIstanbul.covSources.length.should.equal(3)
       Object.keys(v8ToIstanbul.toIstanbul()).should.eql(['/webpack/bootstrap', '/src/index.ts', '/src/utils.ts'].map(path.normalize))
+
+      v8ToIstanbul.destroy()
```

Comment:
having a test that verifies that the source map has been destroyed would be good, perhaps you could follow @SimenB's recommendation of setting it equal to `undefined`, and then just assert this case.

Reply:
added a new test which covers this